### PR TITLE
Form fields

### PR DESCRIPTION
**Pull request by frankbreetz**
Adding from fields allowing for input of customer and their email address

### CODE REVIEW

File: plugins/aeroscope.py [hunk 1]
```diff
@@ -26,6 +30,12 @@
 )
 
 
+class AeroForm(Form):
+    company = StringField("Company", [validators.Length(min=4, max=25)])
```
Comment (fritz-astronomer) [inline plugins/aeroscope.py:34]: `organization`, please

File: plugins/aeroscope.py [hunk 2]
```diff
@@ -68,30 +78,55 @@ def get_json_or_clean_str(o: str) -> Union[List[Any], Dict[Any, Any], Any]:
...
+                runpy.run_path(a)
+            date = datetime.datetime.now(datetime.timezone.utc).isoformat()[:10]
+            content = {
+                # "form":form,
+                "company": form.company.data,
+                "email": form.email.data,
+                # "comany": request.form
+                "telescope_version": "aeroscope",
+                "report_date": date,
+                "organization_name": "aeroscope",
```
Comment (fritz-astronomer) [inline plugins/aeroscope.py:103]: organization would go here

File: plugins/aeroscope.py [hunk 3]
```diff
@@ -26,6 +30,12 @@
 )
 
 
+class AeroForm(Form):
+    company = StringField("Company", [validators.Length(min=4, max=25)])
+    email = StringField("Email Address", [validators.Email()])
```
Comment (fritz-astronomer) [inline plugins/aeroscope.py:35]: Why add email?

Comment (frankbreetz) [reply plugins/aeroscope.py:35]: There is a lookup we can do to automatically connect it via salesforce. Company was a fallback, if they weren't a user in Salesforce

Comment (fritz-astronomer) [reply plugins/aeroscope.py:35]: Let's leave it with just `organization` for now

**Review by fritz-astronomer — CHANGES_REQUESTED** [review]
swap plugin inputs to `organization`, `presigned_url`, remove `email`

**Pull request merged**